shared/aux.go

```diff
@@ -569,3 +571,31 @@ func CleanSliceStrings(stringsSlice []string) []string {
 
 	return stringsSlice
 }
+
+func GetInstallCmd(product, installType, nodeType string) string {
```

Comment:
Should this live here or in `product.go` as it is more product based operation?

Reply:
good idea